guides/common/modules/proc_setting-an-alternative-directory-for-remote-execution-jobs-in-push-mode.adoc

```diff
@@ -0,0 +1,51 @@
+[id="setting-an-alternative-directory-for-remote-execution-jobs-in-push-mode_{context}"]
+ifeval::["{context}" == "managing-hosts"]
+= Setting an alternative directory for remote execution jobs in push mode
+
+By default, {Project} uses the `/var/tmp` directory on hosts for remote execution jobs in push mode.
+
+If the `/var/tmp` directory on your host is mounted with the `noexec` flag, {Project} cannot execute remote execution job scripts in this directory.
+Use this procedure to set a new directory for executing remote execution jobs in push mode.
```

Comment:
@adamlazik1 Can you update the introduction here too? I think the same fix as the one you accepted for guides/common/modules/proc_setting-an-alternative-directory-for-remote-execution-jobs-in-pull-mode.adoc should work.

Reply:
Sorry for the oversight, now fixed. However; I am not going to touch Ansible since the same argument with `noexec` flag does not apply and I am just leaving what was already there.